test/stresses.jl

```diff
@@ -12,7 +12,7 @@ include("testcases.jl")
         Si = ElementPsp(silicon.atnum, psp=load_psp(silicon.psp))
         atoms = [Si => silicon.positions]
         model = model_DFT(lattice, atoms, [:lda_x, :lda_c_vwn]; symmetries)
-        kgrid = [2, 2, 1]
+        kgrid = [3, 3, 1]
```

Comment:
Since it should now work with MPI, can you disable that check in runtests? It'll make sure the CI runs this under MPI.

Reply:
Should I also move `include("stresses.jl")` to e.g. right below `include("forces.jl")`? Or is there a specific reason for the organization?

Reply:
yes please. we are not very organised in runtests.jl

Reply:
Okay I also changed the organization.

Reply:
thanks!